internal/installation/receipt/receipt.go

```diff
@@ -42,6 +43,16 @@ func Load(path string) (index.Receipt, error) {
 	return indexscanner.ReadReceiptFromFile(path)
 }
 
+// CanonicalName returns the index and plugin name from a receipt.
+// The index name is omitted if it is the default index.
+func CanonicalName(receipt index.Receipt) string {
```

Comment:
something defined on the type could be more idiomatic.

`func (r index.Receipt) CanonicalName() string`